.babelrc

```diff
@@ -0,0 +1,11 @@
+{
+  "presets": [
+    "@babel/preset-env",
+    [
+      "@babel/preset-react",
+      {
+        "runtime": "automatic"
+      }
+    ]
+  ]
+}
```

Comment:
Babel is a toolchain that is mainly used to convert new Javascript-style code (the proper term for this is ES5 and ES6 IIRC) into a backwards compatible version of JavaScript. Usually this is done to make code work in old browsers, but in this case it's necessary for our code tests.